README.md

```diff
@@ -789,6 +790,23 @@ $ git push
 
 The behavior of the other modes of ```git push``` is described in the doc of push.default.
 
+<a name="i-want-to-set-a-remote-branch-as-the-upstream-for-a-local-branch"></a>
+### I want to set a remote branch as the upstream for a local branch
+
+You can set a remote branch as the upstream for the current local branch using:
+
+```sh
+$ git branch --set-upstream-to [remotename]/[branch]
+$ or, using the shorthand:
```

Comment:
This line should be in a `#` or outside of the codeblock.

Reply:
OH! I am extremely sorry, this should be in a bash comment indeed. I have fixed it now.